.github/workflows/CondaLock.yml

```diff
@@ -50,51 +50,66 @@ jobs:
            activate-environment: condalock
            use-only-tar-bz2: true # IMPORTANT: This needs to be set for caching to work properly!
 
+      - name: Specify which architectures to build lockfiles for
+        run: |
+          # space delimited list of architectures to send to conda-lock
+          echo "BUILD_ARCHS='64 aarch64 ppc64le'" > ${GITHUB_ENV}
+
       # Could run as single step in parallel, but would complicate logs...
       - name: Run conda-lock base-notebook
         timeout-minutes: 5
         continue-on-error: true
         run: |
           conda clean --all --yes --force-pkgs-dirs
           cd base-notebook
-          conda-lock lock --mamba -k explicit -f environment.yml -p linux-64
-          ../generate-packages-list.py conda-linux-64.lock > packages.txt
+          for ARCH in ${BUILD_ARCHS}; do
+            conda-lock lock --mamba -k explicit -f environment.yml -p linux-${ARCH}
```

Comment:
this might be a good time to switch to a unified lockfile and actually add conda-lock to the `base` environment. (Our current usage of conda-lock pre-dates the unified lockfile format now shown here https://github.com/conda-incubator/conda-lock#basic-usage). Installing the environment with conda-lock would actually recognize the host platform and choose the correct set of packages:

```
conda-lock lock -f environment.yml -p linux-64 aarch64
conda-lock install -n pangeo-notebook
```

the drawback is that if someone just has conda or mamba installed they dont recognize the new unified lockfile format... so you could not do `conda env create -f conda-lock.yml` ...

Reply:
@scottyhq ok that's pretty cool and we clearly should just move to that.

Reply:
Thinking that we might need to update from conda-lock=1.4 to conda-lock=2.x first at https://github.com/pangeo-data/pangeo-docker-images/blob/554675c5269c1b8d634004f6a5529c4031fdeeb4/environment-condalock.yml#L6

Also FYI, `mamba` doesn't support installing from the unified `conda-lock.yml`, but `micromamba` does https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html#conda-lock-yaml-spec-files. See also https://github.com/mamba-org/mamba/issues/1884#issuecomment-1245170833